internal/apps/dop/providers/issue/dao/issue_property.go

```diff
@@ -75,6 +77,41 @@ func (client *DBClient) GetIssueProperties(req pb.GetIssuePropertyRequest) ([]Is
 		return nil, err
 	}
 
+	// propertiesProject 项目级 properties
+	var propertiesProject []IssueProperty
+	overlayIndex := make([]int, 0)
+	if req.ScopeType != "project" {
```

Comment:
do not use literal string, use `apistructs.ProjectScope`.

Reply:
ok,thank you for your suggestion